domains/shockbs.json

```diff
@@ -5,6 +5,6 @@
     },
     "record": {
         "A": ["69.30.249.53"],
-        "MX": ["mail.is-a.dev"]
+        "CNAME": ["proxy.private.danbot.host"]
```

Comment:
you can use CNAME with other records

Reply:
what do you mean

Reply:
> what do you mean

sorry, wrote wrong, you can't use CNAME with other records

Reply:
you should remove the A record if you want to use the CNAME

Reply:
ok